jumanji/environments/logic/rubiks_cube/env_viewer.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Sequence, Tuple
+
+import jax.numpy as jnp
+import matplotlib
+from matplotlib import pyplot as plt
+
+import jumanji.environments
+from jumanji.environments.logic.rubiks_cube.constants import Face
+from jumanji.environments.logic.rubiks_cube.types import State
+
+
+class RubiksCubeViewer:
```

Comment:
After #99 is merged, we can remove this abstract viewer and just keep one concrete viewer that we may want to call `RubiksCubeViewer`.

Reply:
OK but then what would the type hint of the env viewer in the main env file be?

Reply:
It would be `Optional[Viewer[State]]`